server/webapp/WEB-INF/rails.new/app/assets/stylesheets/new-theme.scss

```diff
@@ -3070,12 +3070,6 @@ button.edit-environment {
   padding:   0;
 }
 
-.buildoutput_pre * {
```

Comment:
yeah, all those styles were duplicated elsewhere, and this selector's weight was overriding the simple class selector I used for duration elements, and effectively clobbered the `font-size` rule I was setting.